frontend/src/concepts/pipelines/content/createRun/RunForm.tsx

```diff
@@ -44,9 +45,30 @@ const RunForm: React.FC<RunFormProps> = ({ data, onValueChange }) => (
       value={data.pipeline}
       onChange={(pipeline) => {
         onValueChange('pipeline', pipeline);
+        onValueChange('version', pipeline.default_version);
         onValueChange(
           'params',
-          (pipeline.parameters || []).map((p) => ({ label: p.name, value: p.value ?? '' })),
+          (pipeline.default_version.parameters || pipeline.parameters || []).map((p) => ({
+            label: p.name,
+            value: p.value ?? '',
+          })),
+        );
+      }}
+    />
+    <PipelineVersionSection
```

Comment:
@yannnz Do we still have the Pipeline version dropdown enabled when we only have 1 version(i.e., the Default one) and that is selected by default? Here's an example. 
<img width="665" alt="Screenshot 2023-12-05 at 6 17 53 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/01a2ccf6-987b-4414-a465-38d64256df6a">